### ISSUE TITLE
Cannot start container

### ISSUE THREAD

**Issue by fredericmazzini** [francelabs/datafari-docker#2]
Hi,

I'm trying to test Datafari with your docker image datafari/datafariv4. When I execute the command  

`docker run -t -i -p 8084:8080 -p 8983:8983 datafari/datafariv4 `

I get the following error :

> Fail to reach Elasticsearch on localhost:9200
> Please check your network connection and, in case a proxy is configured, that a proxy exception exists for 'localhost' and '127.0.0.1' !

What's wrong ?

Thanks

Fred

**Comment by otavard** [comment]
Hi,

I just updated the Dockerfile to fix your issue.
It seems that the verification of the ES connection is done too early ie your ES instance is not yet started and so the verification fails.
I increased the sleep time before the verification is done in the Dockerfile. Maybe your container had not enough available resources on the host when you started Datafari container.
Tell me if it is ok now.
Thanks,
Olivier
